Workbooks/WorkbooksMetadata.json

```diff
@@ -701,7 +740,47 @@
     "subtitle": "",
     "provider": "Edoardo Gerosa"
   },
-  {    "workbookKey": "GitHubSecurityWorkbook",
+  {
+    "workbookKey": "VMwareCarbonBlackWorkbook",
+    "logoFileName": "vmwarecarbonblack_logo.svg",
+    "description": "Gain extensive insight into VMware Carbon Black Cloud - Endpoint Standard by analyzing, collecting and correlating Event logs.\nThis workbook provides visibility into Carbon Black managed endpoints and identified threat event",
+    "dataTypesDependencies": [ "CarbonBlackEvents_CL","CarbonBlackNotifications_CL","CarbonBlackAuditLogs_CL" ],
+    "dataConnectorsDependencies": [ "VMwareCarbonBlack" ],
+    "previewImagesFileNames": [ "VMwareCarbonWhite.png", "VMwareCarbonBlack.png" ],
+    "version": "1.0",
+    "title": "VMware Carbon Black",
+    "templateRelativePath": "VMwareCarbonBlack.json",
+    "subtitle": "",
+    "provider": "VMware"
+  },
+  {
+    "workbookKey": "ProofPointTAPWorkbook",
+    "logoFileName": "proofpointlogo.svg",
+    "description": "Gain extensive insight into Proofpoint Targeted Attack Protection (TAP) by analyzing, collecting and correlating TAP log events.\nThis workbook provides visibility into message and click events that were permitted, delivered, or blocked",
+    "dataTypesDependencies": [ "ProofPointTAPMessagesBlocked_CL", "ProofPointTAPMessagesDelivered_CL", "ProofPointTAPClicksPermitted_CL", "ProofPointTAPClicksBlocked_CL" ],
+    "dataConnectorsDependencies": [ "ProofpointTAP" ],
+    "previewImagesFileNames": [ "ProofpointTAPWhite.png", "ProofpointTAPBlack.png" ],
+    "version": "1.0",
+    "title": "Proofpoint TAP",
+    "templateRelativePath": "ProofpointTAP.json",
+    "subtitle": "",
+    "provider": "Proofpoint"
+  },
+  {
+    "workbookKey": "QualysVMWorkbook",
+    "logoFileName": "qualys_logo.svg",
+    "description": "Gain insight into Qualys Vulnerability Management by analyzing, collecting and correlating vulnerability data.\nThis workbook provides visibility into vulnerabilities detected from vulnerability scans",
+    "dataTypesDependencies": ["QualysHostDetection_CL"],
+    "dataConnectorsDependencies": [ "QualysVulnerabilityManagement" ],
+    "previewImagesFileNames": [ "QualysVulnerabilityWhite.png", "QualysVulnerabilityBlack.png" ],
```

Comment:
The preview image files for Qualys doesn't match with the file names. please fix - thanks, for fixing other changes requested

Reply:
orrections have been made and changes pushed.